build/gen_stub.php

```diff
@@ -860,6 +1013,27 @@ public function getFunctionEntry(): string {
         }
     }
 
+    public function getOptimizerInfo(): ?string {
+        if ($this->isMethod() && !$this->isFinalMethod()) {
```

Comment:
Func info doesn't support methods at all, so I don't think that isFinalMethod check should be there.

Reply:
I added the extra condition for future-proofing purposes, because my intention is to add support for methods as well. Is Or does this use-case have any blocker issue?